host.py

```diff
@@ -40,15 +38,23 @@ def __str__(self) -> str:
 
 
 class Login(ABC):
+    def __init__(self, hostname: str, username: str) -> None:
+        self._username = username
+        self._hostname = hostname
+        self.host = paramiko.SSHClient()
+        self.host.set_missing_host_key_policy(paramiko.WarningPolicy())
+
+    def vars(self) -> Dict[str, Any]:
+        return {k: v for k, v in vars(self).items() if k not in ['_key', '_password']}
```

Comment:
this is only used for pretty printing the details of the login. For that, either overwrite `__str__()` or add a method (like `def debug_details(self)->str`)

Reply:
Will do